CHANGELOG.md

```diff
@@ -29,6 +30,47 @@
 - [v0.2.0](#v020---20190401)
 - [v0.1.0](#v010---20190112)
 
+## [v1.7.0] - 2021/05/20
+
+### Added
+
+- State files now support environment variable-based templating. decK can
+  substitute the value of an environment variable into an object in the state
+  file. This is useful for avoiding persistent cleartext storage of sensitive
+  values and populating values that vary between similar configurations.
+  [#286](https://github.com/Kong/deck/pull/286)
+- Sort state file objects by name, to ease comparing state files from
+  similarly-configured instances that do not share object IDs.
+  [#327](https://github.com/Kong/deck/pull/327)
+- Added a default timeout to HTTP requests.
+- Implemented `convert` command for converting state files between Kong Gateway
+  and Konnect configuration formats.
+  [#330](https://github.com/Kong/deck/pull/330)
+- Add `--konnect-addr` flag to set Konnect address.
+  [#374](https://github.com/Kong/deck/pull/374)
```

Comment:
```suggestion
- Add `--konnect-addr` flag to set Konnect address. This can be used to target Konnect
data-centers in geographical regions other than the US.
  [#374](https://github.com/Kong/deck/pull/374)
```